cmake/common.cmake

```diff
@@ -93,9 +93,10 @@ macro(_spl_get_google_test)
 endmacro(_spl_get_google_test)
 
 macro(spl_create_component)
+    cmake_parse_arguments(CREATE_COMPONENT "" "LONG_NAME" "" ${ARGN})

```

Comment:
We added the option for the user to define a human readable name for the component. This name can be used in the documentation. Before the component path was used as component name.